server/game_service.py

```diff
@@ -10,19 +10,28 @@
 from server.games.game import Game, GameState, VisibilityState
 from server.matchmaker import MatchmakerQueue
 from server.players import Player
+from server.rating_service.rating_service import RatingService
+from server.rating_service.typedefs import GameRatingSummary
```

Comment:
I couldn't get that to work without introducing reference cycles, since `game.py` needs `GameRatingSummary` and is needed by e.g. `PlayerService`, which is needed by `RatingService`.
Some of these are only imported for typing though. Would you then fall back to providing the names as string instead of the class for typing?

Reply:
Moving all the `Enum`s defined in `game.py` gets rid of the circular imports :)